Makefile

```diff
@@ -59,10 +59,10 @@ else
 endif
 
 test-server-plain:
-	mocha server/test/*.test.js server/test/**/*.test.js
+	mocha --exit server/test/*.test.js server/test/**/*.test.js
```

Comment:
Why did you need to add this?

Reply:
@kavanagh The default behaviour of `mocha ^4` is not to exit when tests finish running (unlike mocha 3). I'd initially bumped mocha as part of a mass dependency bump, but `karma-mocha` doesn't support passing the exit flag, so I reverted to mocha 3. 

Forgot about the flag in Makefile – it doesn't do anything with mocha 3. Shall I remove it or leave it there for a future bump (once `karma-mocha` supports correct behaviour)?

Reply:
Good detail, thanks! I think leave it as is.